src/crypto.proto.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+module.exports = `enum KeyType {
```

Comment:
That would be awesome, but I think it needs to be exported from libp2p-crypto's index.js first.  At the node repl you can do

```
> require('libp2p-crypto/src/crypto.proto')
'enum KeyType {\n  RSA = 0;\n  Ed25519 = 1;\n}\n\nmessage PublicKey {\n  required KeyType Type = 1;\n  required bytes Data = 2;\n}\n\nmessage PrivateKey {\n  required KeyType Type = 1;\n  required bytes Data = 2;\n}'
```

But the tests fail if I use that in the main code.  I'm guessing that webpack, etc don't expose that path in the built module.

I would much prefer to have the protobuf definition in one place to avoid it going stale... If you want to export it and bump the version, I can add a dependency on the new version here to pull it in.